easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.0.9-GCCcore-6.4.0.eb

```diff
@@ -31,6 +31,10 @@ builddependencies = [
     ('binutils', '2.28'),
 ]
 
+dependencies = [
+    ('libjpeg-turbo', '1.5.3')
```

Comment:
@paulmelis Tests are still failing because of adding `libjpeg-turbo` as a dep in this `LibTIFF` easyconfig using `GCCcore/6.4.0` as a toolchain, despite the version fix from 1.5.2 to 1.5.3.

The problem is that `GCCcore/6.4.0` was used as a subtoolchain for both the `2017b` and `2018a` toolchain generation (historical mistake), and that `libjpeg-turbo` 1.5.2 is used as a dep in `2017b` easyconfigs, while 1.5.3 is used in `2018a`.

So, this is beyond fixing, we'll have to refrain from adding `libjpeg-turbo` in this particular easyconfig...

Reply:
Hopefully fixed via d85f4c3...